pytest_tests/testsuites/services/http_gate/test_http_object.py

```diff
@@ -119,3 +126,152 @@ def test_object_put_get_attributes(self, simple_object_size):
                 http_request_path=request,
                 endpoint=self.cluster.default_http_gate_endpoint,
             )
+
+    @allure.title("Test Put/Get with similar attributes")
+    @pytest.mark.parametrize(
+        "object_size",
+        [pytest.lazy_fixture("simple_object_size"), pytest.lazy_fixture("complex_object_size")],
+        ids=["simple object", "complex object"],
+    )
+    def test_put_get_user_headers(self, object_size: int):
```

Comment:
So actually here 10 test cases merged into one. 
Let say we failed at step 5, all consequent steps (cases) will not run.
I propose to decompose the test to small ones.
Also, some steps seems to be very similiras with small diff. Consider parametrization